src/renderer/components/dialogs/About.tsx

```diff
@@ -41,7 +41,11 @@ export function DCInfo(_props: any) {
 
   return (
     <>
-      <h3>Version details:</h3>
+      <h3>Debug Info for System &amp; Selected Account</h3>
+      <p>
+        Local Account Id:{' '}
+        <b>{window.__selectedAccountId || '(no account selected)'}</b>
+      </p>
       <div className='dialog-about__dc-details'>
         <table>
```

Comment:
Usually it's a good idea to use a CSS grid or flexbox rather than a table.

Reply:
it's a debug view we shouldn't spend to much time on it right now.

Reply:
also in this case the info fits well into a table IMO

Reply:
Yeah! Tables were meant for ... tables! So it's actually a perfect use case!